app/build.gradle

```diff
@@ -13,17 +13,16 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = '1.8'
+        sourceCompatibility JavaVersion.VERSION_17
```

Comment:
I think that we need update Java version for Gradle on CI/CD for a successful project build on CI/CD